crypto/src/test/java/org/web3j/crypto/TransactionDecoderTest.java

```diff
@@ -504,4 +505,109 @@ public static RawTransaction createEip4844RawTransaction() {
                 "",
                 BigInteger.valueOf(30000));
     }
+
+    @Test
+    public void testDecoding7702() {
+        final RawTransaction rawTransaction = createEip7702RawTransaction();
+        final Transaction7702 tx7702 = (Transaction7702) rawTransaction.getTransaction();
+
+        final byte[] encoded = TransactionEncoder.encode(rawTransaction);
+        final String hexEncoded = Numeric.toHexString(encoded);
+
+        final RawTransaction decodedRawTx = TransactionDecoder.decode(hexEncoded);
+        assertNotNull(decodedRawTx);
+        assertTrue(decodedRawTx.getTransaction() instanceof Transaction7702);
+
+        final Transaction7702 decodedTx7702 = (Transaction7702) decodedRawTx.getTransaction();
+        assertEquals(tx7702.getChainId(), decodedTx7702.getChainId());
+        assertEquals(tx7702.getNonce(), decodedTx7702.getNonce());
+        assertEquals(tx7702.getMaxPriorityFeePerGas(), decodedTx7702.getMaxPriorityFeePerGas());
+        assertEquals(tx7702.getMaxFeePerGas(), decodedTx7702.getMaxFeePerGas());
+        assertEquals(tx7702.getGasLimit(), decodedTx7702.getGasLimit());
+        assertEquals(tx7702.getTo(), decodedTx7702.getTo());
+        assertEquals(tx7702.getValue(), decodedTx7702.getValue());
+        assertEquals(tx7702.getData(), decodedTx7702.getData());
+        assertIterableEquals(tx7702.getAccessList(), decodedTx7702.getAccessList());
+        assertIterableEquals(tx7702.getAuthorizationList(), decodedTx7702.getAuthorizationList());
```

Comment:
please export these common lines in a method

Reply:
to push back a little bit on this, the other tests have these assertions repeated between the signed vs unsigned txn tests, so I thought it would be good to follow the existing pattern.

I also generally adhere to the principal that repetition in tests is preferable to adding too many layers of abstraction in the name of DRY: https://medium.com/@maksimovic.danijel989/you-should-repeat-yourself-when-writing-tests-e82b3b04afe6

wdyt? in the end, happy to defer to you as code owner!